sklearn/svm/classes.py

```diff
@@ -867,13 +867,13 @@ class OneClassSVM(BaseLibSVM):
     Attributes
     ----------
     support_ : array-like, shape = [n_SV]
-        Index of support vectors.
+        Indexes of support vectors.
```

Comment:
I hopefully fixed it now in the whole file. It's my first time editing directly on github and not using a separate checkout and editor.